Looms.PoS/Looms.PoS/Controllers/BusinessesController.cs

```diff
@@ -3,6 +3,7 @@
 using Looms.PoS.Application.Features.Business.Commands.UpdateBusiness;
 using Looms.PoS.Application.Features.Business.Queries.GetBusiness;
 using Looms.PoS.Application.Features.Business.Queries.GetBusinesses;
+using Looms.PoS.Application.Models.Responses;
```

Comment:
Remove

Reply:
fixed in "fixed indentation/layout problems, changed Variation to Variation request and made create and update requests take IEnumerable<VariationRequest>"